src/css/angular-tooltips.css

```diff
@@ -1,3 +1,11 @@
+@keyframes fadeIn {
```

Comment:
Great, just call them 720kbFadeIn and 720kbFadeOut just to not make them to collide with other keyframes animations that people could define somewhere else